File: echo.php
<?php

require 'erl_codec.php';

$gateway = Erlang\Gateway::getInstance();
while (true) {
    $message = $gateway->recv();
    $message->data[0] = new Erlang\Atom('py_echo');
    $message->data[1] += 1;
    $gateway->send($message);
}

File: echo.py
#!/usr/bin/env python

from erl_codec import Gateway, Atom

def main():
    gateway = Gateway()
    while True:
        source, data = gateway.recv()
        data += 1
        gateway.send((Atom('php_echo'), data))

if __name__ == '__main__':
    main()
